src/modules/contracts/contracts.controller.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  UseGuards,
+  Req,
+} from '@nestjs/common';
+import { JwtAuthGuard } from 'src/modules/auth/guards/jwt-auth.guard';
+import { ContractsService } from 'src/modules/contracts/contracts.service';
+import { CreateContractDto } from 'src/modules/contracts/dto/create-contract.dto';
+import { UpdateContractDto } from 'src/modules/contracts/dto/update-contract.dto';
+import { AuthRequest } from 'src/utils/@types/AuthRequest';
+
+@Controller('contracts')
+export class ContractsController {
+  constructor(private readonly contractsService: ContractsService) {}
+
+  @UseGuards(JwtAuthGuard)
+  @Post()
+  create(@Body() createContractDto: CreateContractDto) {
+    return this.contractsService.create(createContractDto);
+  }
+
+  @UseGuards(JwtAuthGuard)
+  @Get()
+  findAll() {
```

Comment:
return types for each functions

Reply:
Should we have explicit return types for every controller? I believe all of them are being infered as you can see below:
![image](https://user-images.githubusercontent.com/61670871/189725763-d5e85a3b-e0ad-4539-8eec-911eae262a24.png)

Reply:
Added explicit types for the Contracts controller 👍🏻